pkg/controller/service/reconciler.go

```diff
@@ -104,6 +106,13 @@ func NewServiceReconciler(consoleClient client.Client, config *rest.Config, refr
 		return nil, err
 	}
 
+	go func() {
```

Comment:
I think we should make one synchronous call prior to initializing the service reconciler to eliminate possible race-condition bugs

Reply:
done